skfda/ml/classification/_depth_classifiers.py

```diff
@@ -104,3 +108,120 @@ def predict(self, X):
+        \mathcal{X} &\rightarrow \mathbb{R}^G \\
+        x &\rightarrow \textbf{d} = (D_1^1(x),...,D_g^k(x))
+
+    Where :math:`D_i^j(x)` is the depth of the point :math:`x` with respect to
+    the data in the :math:`i`-th group using the :math:`j`-th depth of the
+    provided list.
+
+    Note that :math:`\mathcal{X}` is possibly multivariate, that is,
+    :math:`\mathcal{X} = \mathcal{X}_1 \times ... \times \mathcal{X}_p`.
+
+    Parameters:
+        depth_methods (default
+            :class:`ModifiedBandDepth <skfda.depth.ModifiedBandDepth>`):
+            List of depth classes to use when calculating the depth of a test
+            sample in a class. See the documentation of the depths module
+            for a list of available depths. By default it is the list
+            containing ModifiedBandDepth.
+
+    Examples:
+        Firstly, we will import and split the Berkeley Growth Study dataset
+
+        >>> from skfda.datasets import fetch_growth
+        >>> from sklearn.model_selection import train_test_split
+        >>> dataset = fetch_growth()
+        >>> fd = dataset['data']
+        >>> y = dataset['target']
+        >>> X_train, X_test, y_train, y_test = train_test_split(
+        ...     fd, y, test_size=0.25, stratify=y, random_state=0)
+
+        >>> from skfda.ml.classification import DDTransform
+        >>> from sklearn.pipeline import make_pipeline
+        >>> from sklearn.neighbors import KNeighborsClassifier
+
+        We classify by first transforming our data using the defined map
+        and then using KNN
+
+        >>> pipe = make_pipeline(DDTransform(), KNeighborsClassifier())
+        >>> pipe.fit(X_train, y_train)
+        Pipeline(steps=[('ddtransform',
+                         DDTransform(depth_methods=[ModifiedBandDepth(),
+                                                    IntegratedDepth()])),
+                        ('kneighborsclassifier', KNeighborsClassifier())])
+
+        We can predict the class of new samples
+
+        >>> pipe.predict(X_test)
+        array([1, 1, 1, 0, 1, 1, 1, 1, 0, 0, 0, 1,
+               1, 0, 0, 0, 0, 1, 1, 1, 1, 0, 1, 1])
+
+        Finally, we calculate the mean accuracy for the test data
+
+        >>> pipe.score(X_test, y_test)
+        0.875
+
+    See also:
+        :class:`~skfda.ml.classification.DTMClassifier`
+        :class:`~skfda.ml.classification.MaximumDepthClassifier`
+
+    References:
+        Li, J., Cuesta-Albertos, J. A., and Liu, R. Y. (2012). DD-classifier:
+        Nonparametric classification procedure based on DD-plot. Journal of
+        the American Statistical Association, 107(498):737-753.
+
+        Cuesta-Albertos, J.A., Febrero-Bande, M. and Oviedo de la Fuente, M.
+        (2017) The DDG-classifier in the functional setting. TEST, 26. 119-142.
+    """
+
+    def __init__(self, depth_methods: List[Depth] = None):
+        if depth_methods is None:
+            self.depth_methods = [ModifiedBandDepth(), IntegratedDepth()]
```

Comment:
Why is this the default? It is not even what the documentation says.

Reply:
Whoops, we should discuss it!